src/Fantomas.Core/Context.fs

```diff
@@ -540,10 +545,11 @@ let ifElse b (f1: Context -> Context) f2 (ctx: Context) = if b then f1 ctx else
 
 let ifElseCtx cond (f1: Context -> Context) f2 (ctx: Context) = if cond ctx then f1 ctx else f2 ctx
 
-let ifStroustrupElse = ifElseCtx (fun ctx -> ctx.Config.ExperimentalStroustrupStyle)
+let ifStroustrupElse =
+    ifElseCtx (fun ctx -> ctx.Config.MultilineBracketStyle = ExperimentalStroustrup)
 
 let ifStroustrup (f1: Context -> Context) =
-    ifElseCtx (fun ctx -> ctx.Config.ExperimentalStroustrupStyle) f1 id
+    ifElseCtx (fun ctx -> ctx.Config.MultilineBracketStyle = ExperimentalStroustrup) f1 id
```

Comment:
I also don't know if `ifStroustrupElse` and `ifStroustrup` are used anywhere. Please remove them if not.